content/shmem_init_thread.tex

```diff
@@ -0,0 +1,55 @@
+\apisummary{
+Initializes the OpenSHMEM library, similar to \FUNC{shmem\_init}, and performs any
+initialization required for supporting the provided thread level.}
+
+%Initializes the \openshmem{} library, similar to \FUNC{shmem\_init}, and in addition performs the initialization required for thread-safe invocation of \openshmem{} functions.}
```

Comment:
Delete these comments before merging, but feel free to leave them in the history.